include/tvm/runtime/packed_func.h

```diff
@@ -293,6 +304,8 @@ class TypedPackedFunc<R(Args...)> {
   friend class TVMRetValue;
   /*! \brief The internal packed function */
   PackedFunc packed_;
+  /*! \brief The name identifying this function */
+  String name_;
```

Comment:
should we just use the `name_` of packed_?

Reply:
Good point, I've made this change.